docs/UserGuide.md

```diff
@@ -3,112 +3,162 @@ layout: page
 title: User Guide
 ---
 
-OfficeHarbor (OH) is a **desktop app for managing the contacts of a tech firm's teams, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, OH can get your contact management tasks done faster than traditional GUI apps.
+# Welcome Message
 
-* Table of Contents
-{:toc}
+Welcome to the user guide of OfficeHarbor (OH). OH is a desktop app designed to assist secretaries of large teach firms in
+efficiently managing the contacts of the tech teams. The app is optimized primarily for keyboard use, allowing for a fast 
+workflow. Additionally, it also has a dynamic on-screen feedback to keep the users informed of their operation throughout 
+the interaction. If you can type fast, OH can get your contact management tasks done faster than traditional mouse-based apps.
+
+## Target Audience
+
+The user guide is mainly intended for secretaries of a large tech firm. We assumed that they will have some experience in
+typing but is unfamiliar with **Command Line Interface (CLI)** and other technical details. 
```

Comment:
From the perspective of the user guide, this line doesn't make that much sense. It seems like it's addressing the CS2103T guidelines more than the user. I'd swap it out for something like "The program is managed almost entirely by commands from keyboard, so a moderately fast typing speed is necessary to be efficient with it. Experience in a command based program is not necessary and is explained in this document."

Reply:
Honestly this has too much overlap with Purpose of this guide. Maybe merge the two somehow.